SettingURLSession.playground/Contents.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+import CryptoKit
+
+class CreatureURL {
+
+    func md5Hash(_ source: String) -> String {
+        return Insecure.MD5.hash(data: source.data(using: .utf8)!).map { String(format: "%02hhx", $0) }.joined()
+    }
+
+    func buildURL(scheme: String, host: String, path: String) -> URL? {
+
+        var components: URLComponents = {
+            var components = URLComponents()
+            components.scheme = scheme
+            components.host = host
+            components.path = path
+            return components
+        }()
```

Comment:
Вычисляемое свойство внутри функции почти выглядит как функция в функции, вообще особо смысла в этом нет, можно было просто записать вот таким образом
```suggestion
            var components = URLComponents()
            components.scheme = scheme
            components.host = host
            components.path = path
```